lib/shared/bucketing-assembly-script/assembly/testHelpers.ts

```diff
@@ -97,8 +97,8 @@ export function doesUserPassRolloutFromJSON(rolloutStr: string | null, boundedHa
     return _doesUserPassRollout(rollout, boundedHash)
 }
 
-export function testConfigBodyClass(configStr: string): string {
-    const config = new ConfigBody(configStr)
+export function testConfigBodyClass(configStr: string, etag: string | null = null): string {
+    const config = new ConfigBody(configStr, etag)
```

Comment:
Could this be...

```suggestion
export function testConfigBodyClass(configStr: string, etag?: string): string {
    const config = new ConfigBody(configStr, etag || null)
```

Reply:
No AS doesn't allow direct optionals like that, you need to use `| null` instead of `?`, and I prefer putting the default values in the param definition.